charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
```

Comment:
Define differing responsibilities of "Project Leads" vs "Product Owners" vs "Science Program Manager".

Reply:
that would be good, a quick definition of these would be helpful, specifically product leads vs. product owners.  Is Trevor a product lead for orange and I'm product owner for example?  We use different terms day to day so that's why I ask.

Reply:
@briandoconnor - Based on the charter process, you are the *Project Lead*. Trevor is the *Product Owner*.

Reply:
@theathorn @briandoconnor - How about:

When a DCP charter is approved, the community explictly delegates authority and leadership for a specific project and its scope to a **Project Lead**. The **Project Lead** is responsible for clearly defining and communicating the long-term vision and direction of the project.

Similar to [Mozilla module owners](https://www.mozilla.org/en-US/about/governance/policies/module-ownership/), **Project Leads** are never *tyrants*. They are chartered to make decisions for their project with input from the community and in the best interests of the community. 

As adapted from the [Scrum process](https://www.scrumguides.org/scrum-guide.html#team-po), the **Product Owner** manages the Product Backlog, including:
>* Clearly expressing Product Backlog items
>* Ordering the items in the Product Backlog to best achieve goals and missions
>* Optimizing the value of the work the Development Team performs
>* Ensuring that the Product Backlog is visible, transparent, and clear to all, and shows what the Scrum Team will work on next
>* Ensuring the Development Team understands items in the Product Backlog to the level needed

The responsibilities of the **Project Lead** and **Product Owner** are tightly intertwined. At an overly simplistic level, the **Project Lead** is more responsible for strategic objectives while the **Product Owner** drives tactical execution. In actual practice, it is a close partnership when planning and coordinating development with other projects.

Reply:
lgtm